### PR TITLE
Add CVE-2015-6420

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
@vasulakkaraju @GeetikaBatra also easy one :wink:

**Comment by vasulakkaraju** [comment]
updated the CVE info. pls review

File: database/java/2015/6420.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - https://h20566.www2.hpe.com/portal/site/hpsc/public/kb/docDisplay?docId=emr_na-c05390722
+    - https://www.kb.cert.org/vuls/id/576313
+    - https://www.tenable.com/security/research/tra-2017-14
+    - https://www.tenable.com/security/research/tra-2017-23
+
+affected:
+    - groupId: commons-collections
+      artifactId: commons-collections
+      version:
+        - "== 3.2.1 & == 4.0"
```
Comment (msrb) [inline database/java/2015/6420.yaml]: Not a valid syntax. Should be:
```
- "<=3.2.1,3.0"
- "==4.0"
```

Comment (msrb) [reply database/java/2015/6420.yaml]: Similarly in `fixedin`

Comment (vasulakkaraju) [reply database/java/2015/6420.yaml]: @msrb as per the ref i could see that  versions 3.2.1 and 4.0 are only affected,any specific reason or ref to update as <=3.2.1,3.0  ?

**Comment by msrb** [comment]
LGTM, thanks :wink:

**Pull request merged**